typescript/infra/src/agents/gcp.ts

```diff
@@ -204,14 +209,23 @@ export function memoryKeyIdentifier(role: string, chainName: string) {
 export async function createAgentGCPKeys(
   environment: string,
   chainNames: string[],
+  validatorCount: number,
+  kathy: boolean = false,
```

Comment:
Good point, originally because we may not want Kathy in certain environments I figured it should be optional. But I see no harm in just creating the key even if it's not used